backend/app/Http/Controllers/Api/V1/DeviceIngestController.php

```
@@ -0,0 +1,44 @@
+
+        $qty = (float) ($request->validated()['qty'] ?? 1);
+
+        $payload = json_encode([
+            'pulse' => $qty,
+            'ts' => now()->timestamp,
+            'source' => 'device',
+        ]);
+
+        // Run synchronously so the device gets an accurate immediate ack and the
+        // count is applied without depending on a queue worker being up.
+        ProcessMqttMessageJob::dispatchSync(
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Jobs\ProcessMqttMessageJob::dispatchSync().


(staticMethod.notFound)